website/management/commands/create_ssrf_tasks.py

```
@@ -0,0 +1,197 @@
+                <ul>
+                    <li>Access internal networks</li>
+                    <li>Read cloud instance metadata</li>
+                    <li>Bypass firewalls</li>
+                    <li>Trigger internal POST requests to sensitive services</li>
+                </ul>
+            """,
+            "mcq_question": "What is the main purpose of an SSRF attack?",
+            "mcq_options": [
+                "A) Trick the server into making unauthorized requests",
+                "B) Steal client-side cookies",
+                "C) Modify HTML content of the user’s browser",
```

Ruff (0.14.3)

String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)